### PR TITLE
fix: vacuum db after log cleanup

### PR DESCRIPTION
**Pull request by LexLuthr**

### CODE REVIEW

File: db/logs.go [hunk 1]
```diff
@@ -72,5 +72,12 @@ func (d *LogsDB) CleanupLogs(ctx context.Context, daysOld int) error {
 	qry := "DELETE from DealLogs WHERE DealUUID IN (SELECT DISTINCT DealUUID FROM DealLogs WHERE CreatedAt < ?)"
 
 	_, err := d.db.ExecContext(ctx, qry, td)
+	if err != nil {
+		return err
+	}
+
+	// Vacuum DealLogs to reduce the log file size. This OP requires free space of "2 * current file size" to succeed
+	qry = "VACUUM DealLogs"
+	_, err = d.db.ExecContext(ctx, qry, td)
```
Comment (dirkmc) [inline db/logs.go:81]: I don't think you need to provide the `td` parameter here.

**Review by dirkmc — COMMENTED** [review]
Please test this out locally before merging.

**Review by dirkmc — COMMENTED** [review]
I'd suggest we don't actually error out if the vacuum command fails. Let's just log an error.

**Comment by LexLuthr** [comment]
I have updated to just log the error instead of erroring out.

**Review by dirkmc — APPROVED** [review]

**Pull request merged**